doc/Roadmap.md

```diff
@@ -5,19 +5,19 @@ The products and services offered by the MongooseIM Platform allow businesses an
 
 ## Current state of the MongooseIM platform
 
-The following components are a part of the MongooseIM platform, not only the XMPP/REST messaging server.
+The following components are all part of the MongooseIM platform, not only the core XMPP/REST messaging server.
```

Comment:
The following components build up the MongooseIM platform, and not only the XMPP/REST server at its core.

?